cmd/bb_browser/browser_service.go

```diff
@@ -49,6 +50,15 @@ func getDigestFromRequest(req *http.Request) (*util.Digest, error) {
 		})
 }
 
+func extractContextFromRequest(req *http.Request) context.Context {
```

Comment:
Could you add a comment above this function to explain what it does/what the purpose is?